setup-tools/generate-cluster-config.sh

```diff
@@ -93,3 +161,8 @@ EOF
 
 scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $KEY_PATH  /tmp/generate-cluster-config.sh /tmp/create-cluster-$CLUSTER_NAME.sh ec2-user@$INSTANCE_IP:~
 ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i $KEY_PATH ec2-user@$INSTANCE_IP "sh ~/generate-cluster-config.sh > /tmp/generate-cluster-config.log && rm -rf ~/generate-cluster-config.sh"
+
+# Clear environment variable
```

Comment:
since we opt to not set, can remove these unset

Reply:
removed, thanks for pointing it out